game/fs.cc

```diff
@@ -196,12 +196,23 @@ Paths getThemePaths() {
 	const fs::path themes = "themes";
 	const fs::path def = "default";
 	const fs::path www = "www";
+	const fs::path js = "js";
+	const fs::path css = "css";
+	const fs::path images = "images";
+
 	std::string theme = config["game/theme"].getEnumName();
 	Paths paths = getPaths();
-	Paths infixes = { themes / theme,
-					  themes / def,
+	Paths infixes = { 
+					  themes / theme,
 					  themes / theme / www,
-					  themes / def / www,
+					  themes / theme / www / js,
+					  themes / theme / www / css,
+					  themes / theme / www / images,
```

Comment:
I've did this in preperation of #251 
I think the themes folder should be specified, and each folder (and thus each theme) should be scanned